docker.test.yml

```diff
@@ -8,10 +8,10 @@ sut:
       brew doctor
       brew tests
       failed=0
-      for i in $$(brew list | egrep -vx 'jdk|magic-blast|tbl2asn|swi-prolog|jdk@8|kat|matplotlib|numpy|portcullis|scipy'); do
+      for i in $$(brew list | egrep -vx 'jdk|magic-blast|tbl2asn|swi-prolog|jdk@8|kat|matplotlib|numpy|portcullis|scipy|emboss|openjdk|protobuf'); do
```

Comment:
Please put this list in alphabetical order. You can remove `jdk` and `jdk@8` once you've confirmed that they're not installed.